### ISSUE TITLE
Decompiling TF2 scenes.image error?

### ISSUE THREAD

**Issue by Ashe-tf** [SlawekNowy/vsif2vcd#11]
After checking the bsp, the program outputs this and closes
```
[I] BSPParser::ExtractNames: Found 0 scene filenames in zi_murky.bsp!
[I] BSPParser::ExtractNames: Found 0 scene filenames in zi_sanitarium.bsp!
[I] BSPParser::ExtractNames: Found 0 scene filenames in zi_woods.bsp!
[I] BSPParser::ExtractNames: Added additional 0 entrypoints for Response Rules.
[I] Program::doStart: Now extracting scene names from response system.
[I] RRParser::initRules: Currently parsing: scripts/talker/response_rules.txt
[I] RRParser::initRules: Succesfully parsed scripts/talker/response_rules.txt
[I] RRParser::dumpSceneNames: Found 0 scene file names from Response System!
[I] Program::appendHardCodedEntries: Appended 71 hardcoded entries.
[I] Program::doStart: Now decompiling...
[I] Program::doStart: CRC (0X00EB69) miss!
[I] Program::doStart: CRC (0X0701AC) miss!
[I] Program::doStart: CRC (0X0D6464) miss!
[I] Program::doStart: CRC (0X188DF2) miss!
[I] Program::doStart: CRC (0X1E051F) miss!
[I] Program::doStart: CRC (0X249483) miss!
[I] Program::doStart: CRC (0X31355C) miss!
[I] Program::doStart: CRC (0X36C55B) miss!
[I] Program::doStart: CRC (0X4F6B4F) miss!
[I] Program::doStart: CRC (0X5A91C0) miss!
[I] Program::doStart: CRC (0X5F7C18) miss!
[I] Program::doStart: CRC (0X61B666) miss!
[I] Program::doStart: CRC (0X6CB834) miss!
[I] Program::doStart: CRC (0X73470C) miss!
[I] Program::doStart: CRC (0X7E39CE) miss!
[I] Program::doStart: CRC (0X7E495E) miss!
[I] Program::doStart: CRC (0X88B7B1) miss!
[I] Program::doStart: CRC (0X9614F9) miss!
[I] Program::doStart: CRC (0X9E52DA) miss!
[I] Program::doStart: CRC (0XA0B898) miss!
[I] Program::doStart: CRC (0XA2ABBD) miss!
[I] Program::doStart: CRC (0XAD16FE) miss!
[I] Program::doStart: CRC (0XADF7B2) miss!
[I] Program::doStart: CRC (0XB0A549) miss!
[I] Program::doStart: CRC (0XB60CDA) miss!
[I] Program::doStart: CRC (0XB72B3B) miss!
[I] Program::doStart: CRC (0XB745E1) miss!
[I] Program::doStart: CRC (0XBD7B39) miss!
[I] Program::doStart: CRC (0XC2EC5E) miss!
[I] Program::doStart: CRC (0XC734E0) miss!
[I] Program::doStart: CRC (0XC90B5D) miss!
[I] Program::doStart: CRC (0XCD9554) miss!
[I] Program::doStart: CRC (0XD8EEE0) miss!
[I] Program::doStart: CRC (0XD91C0A) miss!
[I] Program::doStart: CRC (0XE0AE6E) miss!
[I] Program::doStart: CRC (0XE54E26) miss!
[I] Program::doStart: CRC (0XE7D674) miss!
[I] Program::doStart: CRC (0XEAD826) miss!
[I] Program::doStart: CRC (0XECBF1B) miss!
[I] Program::doStart: CRC (0X106E2A6) miss!
[I] Program::doStart: CRC (0X10EAB18) miss!
[I] Program::doStart: CRC (0X11D3269) miss!
[I] Program::doStart: CRC (0X1224BFD) miss!
[I] Program::doStart: CRC (0X1233056) miss!
[I] Program::doStart: CRC (0X1276253) miss!
[I] Program::doStart: CRC (0X12EC1E6) miss!
[I] Program::doStart: CRC (0X12F89F0) miss!
[I] Program::doStart: CRC (0X1320591) miss!
[I] Program::doStart: CRC (0X1335EBE) miss!
[I] Program::doStart: CRC (0X1399153) miss!
[I] Program::doStart: CRC (0X13AF821) miss!
[I] Program::doStart: CRC (0X13D8B0D) miss!
[I] Program::doStart: CRC (0X1486A80) miss!
[I] Program::doStart: CRC (0X14C13A8) miss!
[I] Program::doStart: CRC (0X1562021) miss!
[I] Program::doStart: CRC (0X1659E70) miss!
[I] Program::doStart: CRC (0X16F125E) miss!
[I] Program::doStart: CRC (0X170D48D) miss!
[I] Program::doStart: CRC (0X176FD9B) miss!
[I] Program::doStart: CRC (0X1771F8A) miss!
[I] Program::doStart: CRC (0X17B3788) miss!
[I] Program::doStart: CRC (0X18082B4) miss!
[I] Program::doStart: CRC (0X18E927C) miss!
[I] Program::doStart: CRC (0X197749E) miss!
[I] Program::doStart: CRC (0X1AE52A6) miss!
[I] Program::doStart: CRC (0X1B53EAC) miss!
[I] Program::doStart: CRC (0X1B6AB40) miss!
[I] Program::doStart: CRC (0X1BC9833) miss!
[I] Program::doStart: CRC (0X1BCEB91) miss!
[I] Program::doStart: CRC (0X1C8C1FE) miss!
[I] Program::doStart: CRC (0X1CF15F6) miss!
[I] Program::doStart: CRC (0X1D30947) miss!
[I] Program::doStart: CRC (0X1E38EF2) miss!
[I] Program::doStart: CRC (0X1E3B804) miss!
[I] Program::doStart: CRC (0X1E3FE62) miss!
[I] Program::doStart: CRC (0X1EE6D44) miss!
[I] Program::doStart: CRC (0X1F9337D) miss!
[I] Program::doStart: CRC (0X1FC9C2E) miss!
[I] Program::doStart: CRC (0X200B744) miss!
[I] Program::doStart: CRC (0X200C4E6) miss!
[I] Program::doStart: CRC (0X207418F) miss!
[I] Program::doStart: CRC (0X2096279) miss!
[I] Program::doStart: CRC (0X20A4E30) miss!
[I] Program::doStart: CRC (0X20D6FDD) miss!
[I] Program::doStart: CRC (0X215A97F) miss!
[I] Program::doStart: CRC (0X21748CA) miss!
[I] Program::doStart: CRC (0X224BCDE) miss!
[I] Program::doStart: CRC (0X228EE6B) miss!
[I] Program::doStart: CRC (0X22AAABA) miss!
[I] Program::doStart: CRC (0X22B284B) miss!
[I] Program::doStart: CRC (0X23174E4) miss!
[I] Program::doStart: CRC (0X23951E2) miss!
[I] Program::doStart: CRC (0X240AD4B) miss!
[I] Program::doStart: CRC (0X2441358) miss!
[I] Program::doStart: CRC (0X2456FA8) miss!
[I] Program::doStart: CRC (0X2471FCC) miss!
[I] Program::doStart: CRC (0X2477A81) miss!
[I] Program::doStart: CRC (0X249664F) miss!
[I] Program::doStart: CRC (0X24C3D13) miss!
[I] Program::doStart: CRC (0X25FA2B7) miss!
[I] Program::doStart: CRC (0X25FD227) miss!
[I] Program::doStart: CRC (0X25FE4D1) miss!
[I] Program::doStart: CRC (0X26F5592) miss!
[I] Program::doStart: CRC (0X2728605) miss!
[I] Program::doStart: CRC (0X285CD86) miss!
[I] Program::doStart: CRC (0X286A4F4) miss!
[I] Program::doStart: CRC (0X28BF44D) miss!
[I] Program::doStart: CRC (0X28E5944) miss!
[I] Program::doStart: CRC (0X28F026B) miss!
[I] Program::doStart: CRC (0X29326B2) miss!
[I] Program::doStart: CRC (0X293D525) miss!
[I] Program::doStart: CRC (0X29DF301) miss!
[I] Program::doStart: CRC (0X29E1728) miss!
[I] Program::doStart: CRC (0X2B5BBF8) miss!
[I] Program::doStart: CRC (0X2BABE73) miss!
[I] Program::doStart: CRC (0X2BEDD6B) miss!
[I] Program::doStart: CRC (0X2C109CE) miss!
[I] Program::doStart: CRC (0X2C30FBA) miss!
[I] Program::doStart: CRC (0X2C76B5D) miss!
[I] Program::doStart: CRC (0X2D9C2A5) miss!
[I] Program::doStart: CRC (0X2DFDEB8) miss!
[I] Program::doStart: CRC (0X2EA7CF4) miss!
[I] Program::doStart: CRC (0X2EEA480) miss!
[I] Program::doStart: CRC (0X2EFE0E9) miss!
[I] Program::doStart: CRC (0X2F04F7D) miss!
[I] Program::doStart: CRC (0X2F6598B) miss!
[I] Program::doStart: CRC (0X2F9C0C2) miss!
[I] Program::doStart: CRC (0X2FDF7F1) miss!
[I] Program::doStart: CRC (0X30127EC) miss!
[I] Program::doStart: CRC (0X3023AA2) miss!
[I] Program::doStart: CRC (0X3036327) miss!
[I] Program::doStart: CRC (0X30B1934) miss!
[I] Program::doStart: CRC (0X30B77EE) miss!
[I] Program::doStart: CRC (0X30CF99C) miss!
[I] Program::doStart: CRC (0X3219AA9) miss!
[I] Program::doStart: CRC (0X3261A0E) miss!
[I] Program::doStart: CRC (0X32A482C) miss!
[I] Program::doStart: CRC (0X32A9D5C) miss!
[I] Program::doStart: CRC (0X32BDEED) miss!
[I] Program::doStart: CRC (0X33C890F) miss!
[I] Program::doStart: CRC (0X348926B) miss!
[I] Program::doStart: CRC (0X35B8AA1) miss!
[I] Program::doStart: CRC (0X36540DF) miss!
[I] Program::doStart: CRC (0X36669BF) miss!
[I] Program::doStart: CRC (0X371C981) miss!
[I] Program::doStart: CRC (0X37F5188) miss!
[I] Program::doStart: CRC (0X380B7D6) miss!
[I] Program::doStart: CRC (0X384E92C) miss!
[I] Program::doStart: CRC (0X38747FE) miss!
[I] Program::doStart: CRC (0X38D6989) miss!
[I] Program::doStart: CRC (0X3900E89) miss!
[I] Program::doStart: CRC (0X3954D59) miss!
[I] Program::doStart: CRC (0X398C856) miss!
[I] Program::doStart: CRC (0X3A33C45) miss!
[I] Program::doStart: CRC (0X3A4D127) miss!
[I] Program::doStart: CRC (0X3B3C96D) miss!
[I] Program::doStart: CRC (0X3B49DF1) miss!
[I] Program::doStart: CRC (0X3B6057B) miss!
[I] Program::doStart: CRC (0X3B6204D) miss!
[I] Program::doStart: CRC (0X3BB5D79) miss!
[I] Program::doStart: CRC (0X3BC6BC2) miss!
[I] Program::doStart: CRC (0X3C2158B) miss!
[I] Program::doStart: CRC (0X3C2651B) miss!
[I] Program::doStart: CRC (0X3C5B24D) miss!
[I] Program::doStart: CRC (0X3CE93FF) miss!
[I] Program::doStart: CRC (0X3D10E4C) miss!
[I] Program::doStart: CRC (0X3D62B47) miss!
[I] Program::doStart: CRC (0X3D6AAC2) miss!
[I] Program::doStart: CRC (0X3DD31A4) miss!
[I] Program::doStart: CRC (0X3DDEAB3) miss!
[I] Program::doStart: CRC (0X3E320CD) miss!
[I] Program::doStart: CRC (0X3EB6433) miss!
[I] Program::doStart: CRC (0X3F0DF4E) miss!
[I] Program::doStart: CRC (0X3F39E9A) miss!
[I] Program::doStart: CRC (0X3F7EC1F) miss!
[I] Program::doStart: CRC (0X3F8DC44) miss!
[I] Program::doStart: CRC (0X40CAE07) miss!
[I] Program::doStart: CRC (0X412A595) miss!
[I] Program::doStart: CRC (0X4136445) miss!
[I] Program::doStart: CRC (0X41B8141) miss!
[I] Program::doStart: CRC (0X41F08A7) miss!
[I] Program::doStart: CRC (0X423A35F) miss!
[I] Program::doStart: CRC (0X42E3D59) miss!
[I] Program::doStart: CRC (0X42E4DC9) miss!
[I] Program::doStart: CRC (0X434CAE7) miss!
[I] Program::doStart: CRC (0X44C52CE) miss!
[I] Program::doStart: CRC (0X456C945) miss!
[I] Program::doStart: CRC (0X45AB7A5) miss!
[I] Program::doStart: CRC (0X460FF3A) miss!
[I] Program::doStart: CRC (0X4613F2E) miss!
[I] Program::doStart: CRC (0X4664C02) miss!
[I] Program::doStart: CRC (0X46B94E1) miss!
[I] Program::doStart: CRC (0X46EB1B2) miss!
[I] Program::doStart: CRC (0X46FEA9D) miss!
[I] Program::doStart: CRC (0X4799E46) miss!
[I] Program::doStart: CRC (0X4812CF3) miss!
[I] Program::doStart: CRC (0X48CF410) miss!
[I] Program::doStart: CRC (0X499FBD7) miss!
[I] Program::doStart: CRC (0X49E2F0B) miss!
[I] Program::doStart: CRC (0X4A41ACD) miss!
[I] Program::doStart: CRC (0X4A5875E) miss!
[I] Program::doStart: CRC (0X4A7B15A) miss!
[I] Program::doStart: CRC (0X4A85D4B) miss!
[I] Program::doStart: CRC (0X4B3D183) miss!
[I] Program::doStart: CRC (0X4B6190B) miss!
[I] Program::doStart: CRC (0X4BD0BE7) miss!
[I] Program::doStart: CRC (0X4BDAFDD) miss!
[I] Program::doStart: CRC (0X4C3E4D3) miss!
[I] Program::doStart: CRC (0X4C6C129) miss!
[I] Program::doStart: CRC (0X4C9811B) miss!
[I] Program::doStart: CRC (0X4CE99E7) miss!
[I] Program::doStart: CRC (0X4D33BA6) miss!
[I] Program::doStart: CRC (0X4DCFEAD) miss!
[I] Program::doStart: CRC (0X4DDA2B1) miss!
[I] Program::doStart: CRC (0X4EDF4F3) miss!
[I] Program::doStart: CRC (0X4F28096) miss!
[I] Program::doStart: CRC (0X4F89F83) miss!
[I] Program::doStart: CRC (0X50270B2) miss!
[I] Program::doStart: CRC (0X5051D52) miss!
[I] Program::doStart: CRC (0X50E5FF0) miss!
[I] Program::doStart: CRC (0X514A54C) miss!
[I] Program::doStart: CRC (0X51C67AF) miss!
[I] Program::doStart: CRC (0X525C7FB) miss!
[I] Program::doStart: CRC (0X535AC11) miss!
[I] Program::doStart: CRC (0X5387951) miss!
[I] Program::doStart: CRC (0X538B5F3) miss!
[I] Program::doStart: CRC (0X53DD602) miss!
[I] Program::doStart: CRC (0X5482E46) miss!
[I] Program::doStart: CRC (0X55186D2) miss!
[I] Program::doStart: CRC (0X551CE21) miss!
[I] Program::doStart: CRC (0X5521310) miss!
[I] Program::doStart: CRC (0X555D4E2) miss!
[I] Program::doStart: CRC (0X560BF55) miss!
[I] Program::doStart: CRC (0X561E445) miss!
[I] Program::doStart: CRC (0X570B802) miss!
[I] Program::doStart: CRC (0X571DBD3) miss!
[I] Program::doStart: CRC (0X572F79A) miss!
[I] Program::doStart: CRC (0X5791A3C) miss!
[I] Program::doStart: CRC (0X579A1C0) miss!
[I] Program::doStart: CRC (0X57E899B) miss!
[I] Program::doStart: CRC (0X5803434) miss!
[I] Program::doStart: CRC (0X5906BFB) miss!
[I] Program::doStart: CRC (0X5A704EF) miss!
[I] Program::doStart: CRC (0X5A9477A) miss!
[I] Program::doStart: CRC (0X5AA9FA0) miss!
[I] Program::doStart: CRC (0X5BBB610) miss!
[I] Program::doStart: CRC (0X5C9B0F0) miss!
[I] Program::doStart: CRC (0X5CA4E67) miss!
[I] Program::doStart: CRC (0X5CB0D6B) miss!
[I] Program::doStart: CRC (0X5CC842C) miss!
[I] Program::doStart: CRC (0X5DAFEA6) miss!
[I] Program::doStart: CRC (0X5E319C7) hit! Path is scenes/expressions/citizen_normal_combat_01.vcd
```

**Comment by JoriKos** [comment]
Mine returns
`[C] Program::doStart: Initalization failed: unknown error`

I can't get it to return anything at all. It seems like this program doesn't work with TF2 at all.

**Comment by MrSoup678** [comment]
Should be fixed in 1.1.1
